lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -11,8 +11,8 @@
             successMessage: function() {
                 return this.indicators.success + interpolate_text(
                     gettext(
-                        'We\'ve sent a confirmation message to {new_email_address}. ' +
-                        'Click the link in the message to update your email address.'
```

Comment:
@symbolist FYI (you were the last one to touch these lines - you can't do this, it doesn't i18n properly. Be sure you verify.)